microservice-app/src/main/java/com/sdevelopment/skt/microservice/dao/ProductRepositoryCustom.java

```diff
@@ -0,0 +1,10 @@
+package com.sdevelopment.skt.microservice.dao;
+
+import com.sdevelopment.skt.common.domain.Product;
+
+import java.util.List;
+
+public interface ProductRepositoryCustom {
```

Comment:
What is the purpose to have these two methods separated from ProductRepository? If you need to save a product, you can use the existing methods on ProductRepository, also, if you need a custom method, you can add it in there

Reply:
I will use then the save to call the stored procedure and another to load all the products from the stored procedure.

Reply:
Fixed! Now I have only ProductRepository interface and ProductRepositoryImpl with the two methods that I need.